### DISCUSSION TITLE
Request to add arrow icon that matches the weight of and style of the 'Selected' icon

### DISCUSSION THREAD

**Discussion by margitkuhlberg** [twilio-labs/paste#3041] [Request: Icon] [answered]
Hey Team! 

My team is developing a table for supplier portal where we would need to use arrow icons and selected icon along the rate brackets. The problem is that the weight and style of the arrow icons in Paste do not match with the Selected icon (number 9): 
![image](https://user-images.githubusercontent.com/77267298/220569661-fb6cd708-abba-4596-8b86-0d295521cc2c.png)

Therefore I have two questions regarding this:
1) Shall we request two heavy weight arrow icons to be added to Paste?
![arrowUp](https://user-images.githubusercontent.com/77267298/220570269-2792cdc2-10ad-4c2f-935b-bd6a6011fb93.png)
![arrowDown](https://user-images.githubusercontent.com/77267298/220570313-3e4e86bb-3687-405a-adee-9b718829b8b9.png)

2) The size is already adjustable but could the weight of the arrows be changed as well so we could align the design to Selected icon without the need to add new icons?

**Comment by serifluous** [comment, marked as answer]
Hey @margitkuhlberg ! The Selected icon unfortunately isn't meant to be used in a case like this to mean "success", which is why its weight is a bit out-of-place. You can take a look at our Status pattern, which shows how we recommend showing successes and errors in table data: https://paste.twilio.design/patterns/status

One thing the Status pattern also outlines is whether or not icons need to be added to all statuses: https://paste.twilio.design/patterns/status#pairing-process-statuses-with-plain-text-statuses You could consider removing the Selected icon altogether, and rely on icons _only_ to draw attention to the "bad" deltas (assuming that's what customers mainly need to know).

I've mocked up some options here: https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?node-id=2304%3A416587&t=ge7tJBMg89aTIehN-0 Options 2B and 3 show explorations that include new icons we could add (e.g., "TrendUpIcon", "TrendDownIcon") to support these kinds of statuses in tables.

Let us know if any of these options are preferable for your use case, or if there's something we're not considering about your customers' needs!

Also, out of curiosity:
1. In the "Quality delta" column, for the cells with down arrows, does "5-10%" mean the quality dropped 5-10%? If it is, I'd recommend, adding a negative sign to the numbers to make that super clear.
2. What does "≤ 50%+" mean in the "Rate delta" column? Is it the same as just "≤ 50%"

**Reply by margitkuhlberg** [reply]
Hi @serifluous ! Thank you for getting back to me so quickly. I will discuss the points and suggestions you made with the product designer. 

Regarding your questions:
1. Quality delta is showing the bracket where the quality belongs to. If delta is in 0-5% one week and the second week it's showing an arrow down and 5-10% bracket then this means that quality has gone down to the second bracket. If the quality had gone from 10-25% to 5-10% bucket, then it would be showing an arrow up, etc. Hope this helps to explain. 

2. It means the same indeed. It means that rate delta is off over 50%.

**Comment by margitkuhlberg** [comment]
Hi @serifluous, we have chosen to go with Option 3 from your recommendations. Thank you for your input!

**Comment by margitkuhlberg** [comment]
@serifluous to go with the 3rd option, we'd need the "TrendUpIcon", "TrendDownIcon". What's the best way to proceed to add those?

**Reply by serifluous** [reply]
Great! Can you submit a request for the icons here? https://airtable.com/shrvMvL3BG6Zm0f9l It'll help us better track & make progress on dependencies.

**Reply by margitkuhlberg** [reply]
Thank you @serifluous ! I have made a request in Airtable ;)

**Reply by margitkuhlberg** [reply]
Hey @serifluous ! Is there a way to track the progress of my request? What would be the possible ETA?

**Reply by serifluous** [reply]
Just tagged you in the ticket! https://issues.corp.twilio.com/browse/DSYS-3836 It's in our current sprint, on track to complete by March 22.

We're still working through our process in communicating dependency statuses so thanks for bearing with us. 🙏

**Comment by nkrantz** [comment]
Hey @margitkuhlberg, feel free to follow along with this PR (https://github.com/twilio-labs/paste/pull/3095). When it merges, all of the new icons will be available for use!

**Reply by margitkuhlberg** [reply]
Thank you @nkrantz and @serifluous for the help! I see they are now available for use in Paste :)
